bitnami/prometheus-operator/README.md

```diff
@@ -226,7 +226,9 @@ The following table lists the configurable parameters of the Prometheus Operator
 | `prometheus.volumes`                                  | Volumes allows configuration of additonal volumes. Evaluated as a template                                                     | `[]` See [docs](https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#prometheusspec) for details                                                                                                                          |
 | `prometheus.volumeMounts`                             | VolumeMounts allows configuration of additional VolumeMounts. Evaluated as a template                                          | `[]` See [docs](https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#prometheusspec) for details                                                                                                                          |
 | `prometheus.additionalPrometheusRules`                | PrometheusRule defines recording and alerting rules for a Prometheus instance.                                                 | `[]`                                                                                                                                                                                                                                                |
-| `prometheus.additionalScrapeConfigsExternal`          | Enable additional scrape configs that are managed externally to this chart                                                     | `false` See [docs](#additional-scrape-configurations) for details.                                                                                                                                                                                  |
+| `prometheus.additionalScrapeConfigsExternal.enabled`  | Enable additional scrape configs that are managed externally to this chart                                                     | `false` See [docs](#additional-scrape-configurations) for details.                                                                                                                                                                                  |
+| `prometheus.additionalScrapeConfigsExternal.name`     | Enable additional scrape configs that are managed externally to this chart                                                     | `fnil`                                                                                                                                                                                                                                              |
```

Comment:
`fnil ` -> `nil`

Reply:
Also, I think it would be a good idea to change the description to something that better suits this specific property. WDYT?